src/test/cypress/support/pageobjects/exercises/text/TextEditorPage.ts

```diff
@@ -23,30 +23,22 @@ export class TextEditorPage {
     }
 
     shouldShowExerciseTitleInHeader(exerciseTitle: string) {
-        cy.get('jhi-header-participation-page').contains(exerciseTitle).should('be.visible');
+        cy.get('#participation-header').contains(exerciseTitle).should('be.visible');
     }
 
     shouldShowProblemStatement() {
-        cy.get('[jhitranslate="artemisApp.exercise.problemStatement"]').should('be.visible');
+        cy.get('#problem-statement').should('be.visible');
     }
 
     getHeaderElement() {
         return cy.get('jhi-header-participation-page');
```

Comment:
This isnt an id.

Reply:
Correct this method isn't used in a test anymore, so I missed it

Reply:
I removed it